client/app/components/composites/Branding/Branding.css

```diff
@@ -0,0 +1,10 @@
+.brandingContainer {
+  border-top: 1px solid #EBEBEB;
```

Comment:
Also: 
app/components/composites/Branding/Branding.css
 2:25  ✖  Expected "#EBEBEB" to be "#ebebeb"   color-hex-case